README.md

```diff
@@ -2,40 +2,12 @@
 
 # Project Name
 
-> Description the project.
+- Hello Microvers
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
```

Comment:
- [ ]  you should use this template and instead of deleting all the section try to customize it to your own project and add your social media links also install a section that tells how you should install this project locally